backend/lib/models/Notification.js

```diff
@@ -51,6 +51,7 @@ const notificationSchema = new Schema(
       type: {
         type: String,
       },
+      verified: Boolean,
```

Comment:
@mannykary please help check if this new field will be handled by our notification services ❔

Reply:
@ddveloper  Do we only want emails to send if verified is true? If yes, we'll need to update the query in the DatabaseHelper to filter for verified: true. It might help if it's indexed.

Reply:
I don't think that's what we want to do. maybe we only need to display the verified check mark ✅ next to the triggeredBy username.
verified will be true if triggeredBy is verified.

Reply:
Plan to merge this PR today, will add this checkmark as new issue if needed.